docs/guides/admin/performance_monitoring.md

```diff
@@ -0,0 +1,106 @@
+# Performance Monitoring
+
+Timesketch includes a built-in performance profiler to help diagnose and troubleshoot performance issues. When enabled, it captures detailed profiling data for every HTTP request to the backend and saves it for later analysis.
+
+## Enabling the Profiler
+
+To enable the performance profiler, you need to edit your `timesketch.conf` file and set the `DEBUG` configuration option to `True`.
+
+1.  **Locate your configuration file:** This is typically at `/etc/timesketch/timesketch.conf`.
+2.  **Enable debug mode:** Find the `DEBUG` setting and change it to `True`.
+
+    ```ini
+    # In /etc/timesketch/timesketch.conf
+    DEBUG = True
+    ```
+
+3.  **Restart the Timesketch server:** For the change to take effect, you must restart the application server.
+
+Once enabled, the profiler will start writing `.prof` files for each request into the `profiles/` directory within your Timesketch project root (e.g., `/usr/local/src/timesketch/profiles`).
```

Comment:
Careful! This folder `/usr/local/src/timesketch/` only exist on local dev deployments using the timesketch-dev container!

Any deployment with a release container (latest or release) will not have this folder. There it would probably best to send the traces to `/var/log/timesketch/` instead since this folder is mapped to the host by default.